.pre-commit-config.yaml

```
@@ -49,6 +49,8 @@ repos:
   rev: 0.6.17
   hooks:
     - id: pip-compile
+      # Workaround for MacOS per https://github.com/vllm-project/vllm/issues/16940
```

The official casing would look better:
```suggestion
      # Workaround for macOS issue https://github.com/vllm-project/vllm/issues/16940
```

Title:
```
[macOS] skip pip-compile in pre-commit on macOS
```